src/v0/destinations/mp/util.js

```diff
@@ -189,7 +190,7 @@ const removeDuplicateMetadata = (mergedBatches) => {
  */
 const buildUtmParams = (campaign) => {
   const utmParams = {};
-  if (campaign) {
+  if (isObject(campaign)) {
```

Comment:
Add a `isDefinedNotNull` check as well

Reply:
Not required. IsObjet will check for null/undefined values also.

Reply:
Have a test case for this in utils.test.js

Reply:
ok